src/Evolution/DgSubcell/Tags/Inactive.hpp

```diff
@@ -27,7 +27,8 @@ struct Inactive : db::PrefixTag, db::SimpleTag {
 
 /// \copydoc Inactive
 template <typename TagList>
-struct Inactive<::Tags::Variables<TagList>> : db::PrefixTag, db::SimpleTag {
+struct Inactive<::Tags::Variables<TagList>>
+    : ::Tags::Variables<db::wrap_tags_in<Inactive, TagList>> {
```

Comment:
@nilsdeppe this tag didn't play nice with the change in this PR because it works differently than all other prefix tags: Other prefix tags rely on `db::add_tag_prefix` for the `Variables` specialization, which produces: `Tags::Variables<tmpl::list<Inactive<TensorTag>>>`. Note the result is a `Tags::Variables` with prefixed _constituents_, i.e. the prefix collapses to the inner tag list. In other words, it does _not_produce: `Inactive<Tags::Variables<tmpl::list<Inactive<TensorTag>>>`.

I understand that it's annoying having to write `db::add_tag_prefix` everywhere, and I have wished to get rid of it for a long time. Do you have an idea how we could handle these Variables prefixes consistently, and also resolve #3229? This change here works ok, but I'm not particularly happy about it.

Reply:
I don't know... This looks like it should be okay. I think the `OnSubcells` can just be used with the `db::add_tag_prefix` instead. The `Inactive` gets used a lot more at the `Variables` even in user-facing code. They could probably both be changed over, but I don't really like the any of anything changing inside subcell code right now because it could quickly result in me having to debug something for a day, which there really isn't time for right now, unfortunately.

You can try sticking this diff on my `compute_dg_time_derivative` branch and if `EvolveValanciaDivCleanMagnetizedTovStar` compiles it should be fine, but even there I'm not sure everything that branch needs to compile has been pushed...

Reply:
See also #1123.

Reply:
I think the alternative is to add a static member variable like `bool add_variables_subitems` that, when present and true, enables the subitems. However, I don't really like to add such a "magic switch" so I thought it's better to bake the subitems feature into `Tags::Variables`, which brings us to the current implementation. @kidder Please let me know when you have thought sufficiently carefully about this and reached a conclusion :)

Reply:
All of the relevant subcell code is merged, so switching to some other interface (like not having the prefixed Variables tags be prefixed themselves, but only the inner tags) would not cause any possibly very difficult merge conflicts.

Reply:
I left this as a possible compile-time optimization and added a comment. For now, no subcell behavior changed. There are more places in the code where we may want to omit the subitems to lighten the DataBox. For example, in the parallel linear solvers many subitems are added to the box but never used.

Reply:
Yea, that's fair :) I think all subcell code would be just fine requiring people to work with the full Variables instead of the individual tensors when dealing with inactive quantities :)